packages/react-switch/src/components/Switch/Switch.types.ts

```diff
@@ -3,69 +3,70 @@ import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utili
 
 export type SwitchSlots = {
   /**
-   * The root of the Switch.
-   */
-  root: Slot<'div'>;
-
-  /**
-   * The bar indicating the status of the Switch.
+   * The root element of the `Switch`.
+   *
+   * The root slot receives the `className` and `style` specified directly on the `<Switch>` tag.
+   * All other native props will be applied to the primary slot: `input`.
    */
-  track: NonNullable<Slot<'div'>>;
+  root: NonNullable<Slot<'div'>>;
 
   /**
-   * The wrapper around the thumb. It is used as the active area for the thumb to position itself.
+   * Hidden input that handles the `Switch's` functionality.
+   *
+   * This is the PRIMARY slot: all native properties specified directly on the `<Switch>` tag will be applied to this
+   * slot, except `className` and `style`, which remain on the root slot.
    */
-  thumbWrapper: NonNullable<Slot<'div'>>;
+  input: NonNullable<Slot<'input'>>;
 
   /**
-   * The circular icon indicating the status of the Switch.
+   * The track over which the thumb slides to indicate the on and off status of the `Switch`.
    */
   thumb: NonNullable<Slot<'div'>>;
 
   /**
-   * The hidden input that handles the Switch's internal functionality.
-   */
-  input: NonNullable<Slot<'input'>>;
-
-  /**
-   * The area in which the Switch's rail allows for the thumb to be dragged.
+   * The circular indicator that moves to visually show the status of the `Switch`.
    */
-  activeRail: NonNullable<Slot<'div'>>;
+  track: NonNullable<Slot<'div'>>;
```

Comment:
is this track required? Could this be done with root, input and thumb? Trying to think of customization options, and the difference between cutomizing a track vs adding styles to the root.

Reply:
I moved the thumb out of the track.